react/Header/NewBadge/NewBadge.less

```diff
@@ -0,0 +1,17 @@
+@import (reference) "~seek-style-guide/theme";
+
+@height: 18px;
+
+.root {
+  content: 'New';
+  display: inline-block;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: @sk-medium;
+  color: @sk-white;
+  background-color: @sk-link;
+  height: @height;
+  line-height: @height;
+  padding: 0 6px;
+  border-radius: @height / 2;
```

Comment:
Should the border radius relate to the height?

Reply:
We discussed this in person, leaving as is.